Dockerfile

```diff
@@ -1,21 +1,31 @@
-FROM golang:1.21 as go-stage
+# syntax=docker/dockerfile:1.3
+
+FROM --platform=$BUILDPLATFORM golang:1.23 as go-stage
 
 WORKDIR /app
 
-COPY go.mod ./
-COPY go.sum ./
+COPY go.mod go.sum ./
 
 RUN go mod download
 
 COPY . ./
 
-RUN go build -o /ical-merger
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=$TARGETARCH go build -ldflags="-s -w" -o /ical-merger .
+
+FROM alpine:3.21
 
-FROM frolvlad/alpine-glibc:latest
+RUN apk update && apk add --no-cache ca-certificates curl && update-ca-certificates && rm -rf /var/cache/apk/*
```

Comment:
@Fesaa if we're using alpine then I'm adding this step to make sure we do a little cleanup as we go.